workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/bpmn/builder/ActivityExpiredNodeBuilder.java

```diff
@@ -26,7 +26,7 @@ public class ActivityExpiredNodeBuilder extends ActivityNodeBuilder {
         builder = context.removeLastEventSubProcessBuilder().subProcessDone();
```

Comment:
changes in folder bpmn/builder  are not related to this PR purpose, just removing warnings for an unused parameter..